### PR TITLE
Create resource responds only with an ID (as JSON)

### LINKED ISSUE #102: Change response of POST /resource to just be the id (as JSON)

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #102

### CODE REVIEW

**Comment by cmh2166** [comment]
Needs to not have state in response, and the spec just needs cleaning up to represent our practices and most recent decisions, but largely this is okay for now and a new PR can handle that.

File: handlers/deposit_resource.go [hunk 1]
```diff
@@ -17,11 +17,9 @@ type depositResourceEntry struct{}
 
 // Handle the delete entry request
 func (d *depositResourceEntry) Handle(params operations.DepositNewResourceParams) middleware.Responder {
-	requestID, _ := uuid.NewRandom()
-
 	// TODO: This should be a DRUID
 	resourceID, _ := uuid.NewRandom()
 
-	response := &models.DepositNewResourceOKBody{RequestID: requestID.String(), State: "deposited", ID: resourceID.String()}
```
Comment (cmh2166) [inline handlers/deposit_resource.go]: no need to pass State: at present.

File: swagger.json [hunk 2]
```diff
@@ -38,14 +38,6 @@
                 "id" : {
                   "type" : "string",
                   "example" : "oo000oo0001"
-                },
-                "state" : {
```
Comment (cmh2166) [inline swagger.json]: remove state for now.

**Comment by aaron-collier** [comment]
+1

**Pull request merged**